data/campaigns/fri04.wmf/scripting/starting_conditions.lua

```diff
@@ -157,8 +157,10 @@ else
    }
 
    hq = p3:place_building("empire_port_large", map.player_slots[3].starting_field, false, true)
+   local gold_iron_ratio = {80, 60, 40}
    hq:set_wares {
-         gold = campaign_data.payment,
+         gold = campaign_data.payment * gold_iron_ratio[difficulty] / 100,
```

Comment:
these lines need to use the lua integer division operator "//" otherwise the game crashes here

Reply:
Fixed
In my last run of fri03 the empire received exactly 300 pieces of gold so I didn't get this bug :D